runtime/arctic/src/lib.rs

```diff
@@ -920,6 +917,7 @@ impl_runtime_apis! {
 				None
 			};
 
+			let is_transactional = false;
```

Comment:
Was this always default to false before?

Reply:
Yes, this is an extra argument needed to satisfy, I have copied it from frontier node template